halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/providers/GetAccountCommandBuilder.java

```diff
@@ -39,6 +39,6 @@ private GetAccountCommand(String providerName) {
     }
 
     @Getter(AccessLevel.PROTECTED)
-    String providerName;
+    private String providerName;
```

Comment:
I'm not familiar with Lombok, but wouldn't `AccessLevel.PROTECTED` overwrite `private`? Or is that the access modifier for the getter function?